res/skins/Shade/style.qss

```diff
@@ -83,34 +84,266 @@ WEffectSelector::checked
+  border: 1px solid #585858;
+  font-weight: normal;
+  color: #9e9e9e;
+  background-color: #0f0f0f;
+  alternate-background-color: #1a1a1a;
+  selection-color: #e6e6e6;
+  selection-background-color: #666;
+}
 
-QPushButton#LibraryBPMButton {background: transparent; border: 0;}
-QPushButton#LibraryBPMButton::checked {image: url(:/images/library/ic_library_checked.png);}
-QPushButton#LibraryBPMButton::!checked {image: url(:/images/library/ic_library_unchecked.png);}
+QTreeView {
+  min-height: 90px;
+  min-width: 90px;
+  border-style: solid;
+  border-color: #585858;
+  show-decoration-selected: 0;
+  /* Don't define top margin here, it won't shift ScrollBar.
+  Defined by SearchBox	*/
+  margin: 0px;
+  }
+  QTreeView:focus,
+  QTableView:focus { /*
+    New Library navigation COs only work if TreeView or TableView have focus.
+    Clicking on buttons, sliders and visuals elsewhere removes focus from Library.
+    In conjuction with [Library],MoveFocusBackward/..Forward, this helps a lot. */
+    border-color: #ff6600;
+  }
+
+/*  Closed branch of tree 	*/
+QTreeView::branch:has-children:!has-siblings:closed,
+QTreeView::branch:closed:has-children:has-siblings {
+  image: url(skin:/style/style_branch_closed.png);
+  }
+  QTreeView::branch:has-children:!has-siblings:closed:selected,
+  QTreeView::branch:closed:has-children:has-siblings:selected {
+    image: url(skin:/style/style_branch_closed_selected.png);
+    background-color: #666;
+  }
+
+/*  Open branch of tree 	*/
+QTreeView::branch:open:has-children:!has-siblings,
+QTreeView::branch:open:has-children:has-siblings {
+  image: url(skin:/style/style_branch_open.png);
+  }
+  QTreeView::branch:open:has-children:!has-siblings:selected,
+  QTreeView::branch:open:has-children:has-siblings:selected {
+    border-image: none;
+    image: url(skin:/style/style_branch_open_selected.png);
+    background-color: #666;
+  }
+  /* space left of selected child item */
+  QTreeView::branch:!has-children:!has-siblings:closed:selected,
+  QTreeView::branch:closed:!has-children:has-siblings:selected,
+  QTreeView::branch:open:!has-children:!has-siblings:selected,
+  QTreeView::branch:open:!has-children:has-siblings:selected {
+    border-image: none;
+    background-color: #0f0f0f;
+  }
+
+QTreeView::item:selected,
+QTableView::item:selected {
+  border-image: none;
+  background-image: none;
+  color: #fff;
+  background-color: #666;
+  }
+
+QTreeView::item:!selected {
+  border-image: none;
+  background-image: none;
+  background-color: #0f0f0f;
+  color: #999;
+  }
+
+  /* BPM lock icon in the library "BPM" column. */
+  #LibraryBPMButton::indicator:checked {
+    image: url(:/images/library/ic_library_checked.png);
+    }
+    #LibraryBPMButton::indicator:unchecked {
+      image: url(:/images/library/ic_library_unchecked.png);
+      }
+  #LibraryBPMButton::item {
+    color: #cfcfcf;
+    }
+    #LibraryBPMButton::item:selected {
+      color: #ededed;
+      background-color: #666;
+      }
+
```

Comment:
I suggest to add hover for BPM Button as well
```    
#LibraryBPMButton::item:hover,
   #LibraryBPMButton::item:checked:hover{
	background: #454545;
}```

Reply:
I see, but @daschuer as the skin designer pointed out: hover effects are rare & subtle in Shade.
For the preview button it makes sense, as it actually sets things in motion (and ejects the previously loaded track), but I agree it's unnecessary for other buttons in Shade